designs/administrator/olm/list-of-operators/list-of-operators.md

```diff
@@ -0,0 +1,57 @@
+---
+parent: Administrator
+version: 4.5
+---
+
+# List of Operators
+
+## All-projects view
+This is the Installed Operators page where “all-projects” has been selected from the project-selector. Info icons have been added next to the Managed Namespaces column to provide the user more information about what these column titles mean.The user can sort the column via the sort icon (which looks like an arrow). There are a few additions to the flow:
+
+In the Managed Namespaces column there are three different ways the Namespace information can appear. In the Managed Namespaces column there are three different ways the Namespace information can appear.
+
+  a. “My-project” is representative of a single Namespace.
+
+  b. “All Namespaces” is a text only link that is associated with the Namespaces that fall across all Namespaces.
```

Comment:
Should we use regular black text for this rather than what looks like grayed out/disabled text?

Reply:
Agreed that this should probably be black, based on looking at it in a 4.5 cluster. Perhaps bug worthy.

Of note is that the seemingly equivalent interaction on the Role Bindings page just uses black text reading "all". Maybe one or the other should change to be consistent?

![all ns](https://user-images.githubusercontent.com/54860125/80754550-38a64080-8afd-11ea-8789-8d09745f65bb.png)

Reply:
@lizsurette and @itsptk I'm going to take note of these things. I agree it is bug worthy.

Reply:
Sounds good!